aviary/examples/multi_mission/run_multimission_example_large_single_aisle.py

```diff
@@ -0,0 +1,328 @@
+aircraft:crew_and_payload:num_first_class,0,unitless
+
+"""
+import copy as copy
+from aviary.examples.example_phase_info import phase_info
+from aviary.variable_info.variables import Mission, Aircraft, Settings
+from aviary.variable_info.enums import ProblemType
+import aviary.api as av
+import openmdao.api as om
+import matplotlib.pyplot as plt
+from os.path import join
+import numpy as np
+import dymos as dm
+import warnings
+import sys
+from aviary.subsystems.mass.flops_based.furnishings import TransportFurnishingsGroupMass
+from aviary.api import SubsystemBuilderBase
+from aviary.validation_cases.validation_tests import get_flops_inputs
+
+# fly the same mission twice with two different passenger loads
+phase_info_primary = copy.deepcopy(phase_info)
+phase_info_deadhead = copy.deepcopy(phase_info)
+# get large single aisle values
+aviary_inputs_primary = get_flops_inputs('LargeSingleAisle2FLOPS')
+aviary_inputs_primary.set_val(Mission.Design.GROSS_MASS, val=100000, units='lbm')
+aviary_inputs_primary.set_val(Settings.VERBOSITY, val=1)
+
+aviary_inputs_deadhead = copy.deepcopy(aviary_inputs_primary)
+aviary_inputs_deadhead.set_val('aircraft:crew_and_payload:num_passengers', 1, 'unitless')
+aviary_inputs_deadhead.set_val(
+    'aircraft:crew_and_payload:num_tourist_class', 1, 'unitless')
+aviary_inputs_deadhead.set_val(
+    'aircraft:crew_and_payload:num_business_class', 0, 'unitless')
+aviary_inputs_deadhead.set_val(
+    'aircraft:crew_and_payload:num_first_class', 0, 'unitless')
+
+Optimizer = 'SLSQP'  # SLSQP or SNOPT
+
+
+class MultiMissionProblem(om.Problem):
+    def __init__(self, aviary_values, phase_infos, weights):
+        super().__init__()
+        self.num_missions = len(aviary_values)
+        # phase infos and aviary_values length must match - this maybe unnecessary if
+        # different aviary_values (payloads) fly same mission (say pax vs cargo)
+        # or if same payload flies 2 different missions (altitude/mach differences)
+        if self.num_missions != len(phase_infos):
+            raise Exception("Length of aviary_values and phase_infos must be the same!")
+
+        # if fewer weights than aviary_values are provided, assign equal weights for all aviary_values
+        if len(weights) < self.num_missions:
+            weights = [1]*self.num_missions
+        # if more weights than aviary_values, raise exception
+        elif len(weights) > self.num_missions:
+            raise Exception("Length of weights cannot exceed length of aviary_values!")
+        self.weights = weights
+        self.phase_infos = phase_infos
+
+        self.group_prefix = 'group'
+        self.probs = []
+        self.fuel_vars = []
+        self.phases = {}
+        # define individual aviary problems
+        for i, (aviary_values, phase_info) in enumerate(zip(aviary_values, phase_infos)):
+            prob = av.AviaryProblem()
+            prob.load_inputs(aviary_values, phase_info)
+            prob.check_and_preprocess_inputs()
```

Comment:
group_1 (the deadhead mission) mission passenger counts are getting reset to design passenger counts by the preprocessor

Reply:
The example included in this PR shows as-flown passengers set to 1 for the deadhead mission:
`aviary_inputs_deadhead.set_val(Aircraft.CrewPayload.NUM_PASSENGERS, 1, 'unitless')` . 

Setting to zero requires more finagling for the time being as I documented in the DOCS you'll need to set `TOTAL_PAYLOAD_MASS` to get that to work.